azure-pipelines.yml

```diff
@@ -130,13 +131,18 @@ jobs:
                                 'PROJ_HASH':'${PROJ_HASH}'\
                             }, "
 
+              # Some debugging info
+              echo " ---> ${NAME}: ${BB_HASH}/${PROJ_HASH}"
```

Comment:
Can you also print `TREE_HASH` and `META_HASH`?

Reply:
Sure, might as well debug all the things.